testing/testing.go

```diff
@@ -110,3 +118,56 @@ func EnsureNodeSync(nodeManager common.NodeManager) {
 		}
 	}
 }
+
+// GetRemoteURLFromID returns asociated network url for giving network id.
+func GetRemoteURLFromID(id int) (string, error) {
```

Comment:
Small digression. This is a great candidate for named return values:
```go
func GetRemoteURLFromID(id int) (url string, err error) {
  switch id {
  case params.MainNetworkID:
    url = params.MainnetEthereumNetworkURL
  }

  err = ErrStatusPrivateNetwork

  return
}
```

Reply:
This function name is a little bit different than below. Shouldn't it be `GetRemoteURLFromNetworkID`?